Dockerfile

```diff
@@ -32,6 +39,6 @@ RUN \
     mv docker /usr/bin/$DOCKER_VERSION && \
     chmod +x /usr/bin/$DOCKER_VERSION
 
-ENV TINI_VERSION v0.16.1
+ENV TINI_VERSION v0.18.0
```

Comment:
not sure about `tini`
is it better to keep it in the end repositories or in docker-java itself?

Reply:
docker-java is just a java base and can be used anywhere.
This is a base for our micro-services and for that reason they all should use tini.

Reply:
sounds good :+1: